File: Desafios em C#/1013 - O Maior.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1013 {

    static void Main(string[] args) { 

            int MaiorAB,MaiorABC;

            String linha = Console.ReadLine();
            string[] v = linha.Split(' ');
            
            
            int a = int.Parse(v[0]);
            int b = int.Parse(v[1]);
            int c = int.Parse(v[2]);

            MaiorAB = (a + b + Math.Abs(a - b )) / 2;
            MaiorABC = (MaiorAB + c + Math.Abs(MaiorAB - c)) / 2;
            Console.WriteLine(MaiorABC + " eh o maior");
    }

}

File: Desafios em JavaScript/1013 - O Maior.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split(' ');

var MaiorAB, MaiorABC;
        
var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseInt(lines.shift());

MaiorAB = parseInt((a + b + Math.abs(a - b )) / 2);
MaiorABC = parseInt((MaiorAB + c + Math.abs(MaiorAB - c)) / 2);
console.log(`${MaiorABC} eh o maior`);